fonts.css

```diff
@@ -0,0 +1,16 @@
+@font-face {
+  font-family: "GothamPro";
+  src: 
+    url("./fonts/GothamPro-Bold.woff2") format("woff2"),
+    url("./fonts/GothamPro-Bold.woff") format("woff");
+  font-style: normal;
+  font-weight: bold;
+}
+@font-face {
+  font-family: "GothamPro";
+  src: 
+    url("./fonts/GothamPro.woff2") format("woff2"),
+    url("./fonts/GothamPro.woff") format("woff");
+  font-style: normal;
+  font-weight: normal;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/step/pullRequest?prid=5896505)